modules/ROOT/pages/enable-verbose-logging.adoc

```diff
@@ -0,0 +1,68 @@
+= Enable Verbose Logging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can enable verbose logging for different Modules and Connectors. Because of the high impact caused in application performance, verbose logging must be enabled only for testing and troubleshooting purposes.
+
+== Configure Verbose Logging
+
+To configure verbose logging, add an entry to your `log4j2.xml` indicating the package names of the Connectors or Modules you want to log, and set the log level to `DEBUG`.
```

Comment:
To configure verbose logging, add an entry **to  your`log4j2.xml` file,** indicating the package names of the **connectors or modules** you want to log, and set the log level to `DEBUG`.

Reply:
Edit changes indicated by boldface.